SimplenoteWidgets/Supporting Files/SimplenoteWidgetsExtension-Debug.entitlements

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+    <key>previous-application-identifiers</key>
+         <array>
+             <string>4ESDVWK654.com.codality.NotationalFlow</string>
+         </array>
```

Comment:
@mokagio Given your explanation from last comment (which I tend to agree with), shouldn't we remove the key in this `SimplenoteWidgetsExtension-Debug.entitlements` file too, to align with the fact that it's not here in other `SimplenoteWidgetsExtension-<Config>.entitlements` file?

```suggestion
```